source/developers/sts-for-operator.rst

```diff
@@ -0,0 +1,170 @@
+   To use STS with the Operator, you must first explicitly enable it.
+
+How STS Authorization Works in Kubernetes
+-----------------------------------------
+
+An application can use an ``AssumeRoleWithWebIdentity`` call to send a request for temporary credentials to the MinIO Operator.
+The Operator checks the validity of the request, retrieves policies for the application, obtains credentials from the tenant, and then passes the credentials back the application.
+The application uses the issued credentials to work with the object storage on the tenant.
+
+.. image:: /images/k8s/sts-diagram.png
+   :width: 600px
+   :alt: A diagram showing STS token process flow on a Kubernetes MinIO deployment between the requesting application, MinIO Operator, Kubernetes API, PolicyBinding custom resource definition, and the MinIO tenant.
+   :align: center
+
+The complete process includes the following steps:
+
+1. An application sends an API request to the MinIO Operator containing the name of the application and a service account to use.
+2. The MinIO Operator uses the Kubernetes API to check that the JSON Web Token (JWT) associated with the :ref:`service account <minio-operator-sts-service-account>` in the application's request is valid.
+3. The Kubernetes API returns the results of its validity check.
+4. The MinIO Operator checks for a :ref:`Policy Binding <minio-operator-sts-policy-binding>` that matches the application.
+5. The PolicyBinding CRD returns the policy or policies that match the request, if any.
+6. The MinIO Operator sends the combined policy information for the application to the MinIO Tenant.
+7. The tenant creates temporary credentials matching the policy or policies for the request and returns those to the MinIO Operator.
+8. The MinIO Operator forwards the temporary credentials back to the application.
+9. The application uses the credentials to send the object storage calls to the MinIO tenant.
+
+Requirements
+------------
+
+STS for the MinIO Operator requires the following:
+
+- MinIO Operator v5.0.0 or later.
+- The deployment **must** have :ref:`TLS configured <minio-tls>`.
+- :envvar:`OPERATOR_STS_ENABLED` environment variable set to ``on``.
+
+Procedure
+---------
+
+1. Enable STS functionality for the deployment
+   
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl -n minio-operator set env deployment/minio-operator OPERATOR_STS_ENABLED=on
+   
+   - Replace ``minio-operator`` with the namespace for your deployment.
+   - Replace ``deployment/minio-operator`` with the value for your deployment's MinIO Operator.
+
+     You can find the deployment value by running ``kubectl get deployments -n <namespace>``, where you replace ``<namespace>`` with the namespace for the MinIO Operator.
+     Your MinIO Operator namespace is typically ``minio-operator``, though this value can change during install.
+
+2. Ensure an appropriate :ref:`policy <minio-policy>` exists on the MinIO Tenant for the application to use
```

Comment:
```suggestion
2. Ensure an appropriate :ref:`policy <minio-policy>` exists on the MinIO Tenant for the application to use.
   This procedure includes associating that policy to the ServiceAccount through a PolicyBinding object.
```

Reply:
I'll add something to this effect, but I find it kind of awkward because the very next step does this.